docs/book/src/developer/architecture/controllers/control-plane.md

```diff
@@ -234,6 +234,9 @@ The `status` object **may** define several fields:
   exist in the cluster. For example, managed control plane providers for AKS, EKS, GKE, etc, should
   set this to `true`. Leaving the field undefined is equivalent to setting the value to `false`.
 
+Note: once any of `failureMessage` or `failureMessage` surface on the cluster, they cannot be restored anymore (it is
```

Comment:
on the control plane?

Reply:
filureReason/Message  become terminal when they surface on the cluster who is referencing the control plane object (I have updated this and the other comment to clarify this)